### PR TITLE
set source to maven-javadoc-plugin #911

### PR DESCRIPTION
**Pull request by btonokit**
Please review #911

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -272,6 +272,13 @@
                 </container>
               </configuration>
             </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <configuration>
+                <source>${java-version}</source>
+              </configuration>
+            </plugin>
```
Comment (btkatoufm) [inline pom.xml:281]: - [ ] It has been added to `profile travis`, is it correct?

Comment (yoshikawaa) [inline pom.xml:281]: - [ ] Is it enough to use `source` property?

Comment (btonokit) [reply pom.xml]: It was a mistake. Corrected the description location of `maven-javadoc-plugin`

Comment (btonokit) [reply pom.xml]: I think it's enough to use `source` property.
I have confirmed that javadoc can be generated below:
- java 1.8.0_172-b11
- java 11+28
- java 11.0.5+10-LTS

I think that other properties set by gfw should be set originally, and `source` property should be set by the user consciously.

**Review by btkatoufm — APPROVED** [review]

**Pull request merged**